main.nf

```diff
@@ -616,12 +626,12 @@ process bam2fq {
             out1 = name+"_"+params.name1+".unaligned_R1.fastq"
             out2 = name+"_"+params.name1+".unaligned_R2.fastq"
             """
-            bedtools bamtofastq -i $bam -fq $out1 -fq2 $out2
+            samtools fastq -1 $out1 -2 $out2 $bam
```

Comment:
I think you might be missing a re-direct here: `samtools fastq 1.fq 2.fq > $bam

Reply:
I made it now more verbose with c2d4164 , but this is the correct syntax.

Example from the [samtools doc](http://www.htslib.org/doc/samtools.html):
`samtools fastq -1 paired1.fq -2 paired2.fq -0 /dev/null -s /dev/null -n -F 0x900 in.bam`

Reply:
Oh interesting. I assumed you had missed it because the command below does use `>`, but good to know.